pom.xml

```
@@ -89,6 +89,11 @@
       <artifactId>github-branch-source</artifactId>
       <version>${github-branch-source.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.kohsuke</groupId>
```

We need to depend on the [github-api-plugin](https://plugins.jenkins.io/github-api/) instead of the package directly, or users may get `java.lang.NoSuchMethodError: org.kohsuke.github.GHRepository.updateCheckRun(J)Lorg/kohsuke/github/GHCheckRunBuilder` like me.

```suggestion
      <groupId>org.jenkins-ci.plugins</groupId>
```